internal/commands/pull/command.go

```diff
@@ -258,17 +258,22 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 		ui.Print(terminal.NewDebugLog("Fetched hosting assets"))
 	}
 
-	successfulTemplateWrites := make([]string, 0, len(clientTemplates))
+	successfulTemplateWrites := make([]interface{}, 0, len(clientTemplates))
 	for _, ct := range clientTemplates {
 		if err := local.WriteZip(filepath.Join(pathFrontend, ct.id), ct.zipPkg); err != nil {
 			return fmt.Errorf("unable to save template '%s' to disk: %s", ct.id, err)
 		}
-		successfulTemplateWrites = append(successfulTemplateWrites, ct.id)
+		readmeFile := ""
+		if matches, err := filepath.Glob(filepath.Join(pathRelative, local.FrontendPath, ct.id, "README.*")); err == nil && len(matches) == 1 {
+			readmeFile = matches[0]
+		}
+		successfulTemplateWrites = append(successfulTemplateWrites, fmt.Sprintf("[%s] %s", ct.id, readmeFile))
 	}
 
 	ui.Print(terminal.NewTextLog("Successfully pulled app down: %s", pathRelative))
 	if len(successfulTemplateWrites) != 0 {
-		ui.Print(terminal.NewListLog("Successfully saved template(s) to disk", successfulTemplateWrites))
+		ui.Print(terminal.NewListLog("Successfully saved template(s) to disk", successfulTemplateWrites...))
+		ui.Print(terminal.NewTextLog("  Navigate to the saved directory to view directions on how to run the template app(s)"))
```

Comment:
and as a contradiction to what I mentioned earlier with initializing the list with a capacity+1, do we actually want this "Navigate" message to be indented along with the list of templates saved?  from the looks of it, even with just one template saved it sort of gets buried in the list making it hard to see

---

also just calling out the "template" prefix to `app` here so we can remain consistent with the naming style, however we land on it

Reply:
discussed offline: "un-identing" the "Naviagate" message was approved by design